### ISSUE TITLE
prompts and chats do not save

### ISSUE THREAD

**Issue by bluenevus** [MicroHealthLLC/chatbot-ui#52]
after creating a chat on the left and a prompt to reuse on the right of the screen, refreshing the screen or logging back in results in that missing

**Comment by LAlexMHllc** [comment]
Same as Issue #26 and Issue #27

**Comment by b-solution** [comment]
@bluenevus Hi, hope so you are doing well. I believe your connection with firebase is not working as expected. Kindly check your firebase config. If you want your own configuration you can send your's ID or test it on mine database.

**Comment by jldroid19** [comment]
@b-solution we need to setup a database and firebase configuration that will be used in our environment. Can you send over the details needed? Like what type of database you are using and the firebase configuration details needed to make this work?

**Comment by b-solution** [comment]
@jldroid19 Certainly! Could you please provide me with your email address? Once I have it, I will add you to the Firebase database.

**Comment by jldroid19** [comment]
@b-solution here is my email: jacob.lafrance@microheathllc.com.

Question, what account is this firebase database stored in? If it is not stored in a MH company database we need get it moved over to a MH account.

**Comment by b-solution** [comment]
> @b-solution here is my email: [jacob.lafrance@microheathllc.com](mailto:jacob.lafrance@microheathllc.com).
> 
> Question, what account is this firebase database stored in? If it is not stored in a MH company database we need get it moved over to a MH account.

yeah @jldroid19  You can just check here schema and get a idea how db look like after that you can setup you own MH account.

**Comment by jldroid19** [comment]
@b-solution Please work on the MongoDB implementation.

**Comment by LAlexMHllc** [comment]
Prompts are saved to the DB now, but since they do not consistently display on the sidebar, user can not edit/delete prompt. (See Issue #26)

**Comment by LAlexMHllc** [comment]
Chats and Prompts created after update do display on the sidebars after reloading or logging back in. Old chats/prompts that were not deleted before the update, still do not display on the sidebars.

**Comment by jldroid19** [comment]
@LAlexMHllc can you please provide screenshots of this example?

**Comment by LAlexMHllc** [comment]
Screenshot Example of missing prompts from sidebar

![chatbot_missing_prompts_from_sidebar](https://github.com/MicroHealthLLC/chatbot-ui/assets/11938477/dac115e5-ce28-4704-ad7f-0aa3858077fb)

**Comment by b-solution** [comment]
@LAlexMHllc I hope you are well. Can you please test it again and let me know if it's not working good.

**Comment by LAlexMHllc** [comment]
Issue still exists for prompts saved BEFORE the update, therefore user is unable to edit or delete these prompt.
Screenshot Example

![prompts_prior_to_update_still_do_not_display_on_sidebar](https://github.com/MicroHealthLLC/chatbot-ui/assets/11938477/3dfb57d1-e6d7-445f-8804-8df7ad68cfba)
